source/agora/node/FullNode.d

```diff
@@ -453,9 +453,7 @@ public class FullNode : API
     /// GET: /preimage
     public override PreImageInfo getPreimage (Hash enroll_key)
     {
-        PreImageInfo preimage;
-        this.enroll_man.getValidatorPreimage(enroll_key, preimage);
-        return preimage;
+        return this.enroll_man.getValidatorPreimage(enroll_key);
```

Comment:
Technically here we should throw an Exception for the generator to pick up a non-200 is needed. But that works too.

Reply:
Hmm. It seems then we need to be more consistent. `getEnrollment` / `getMerklePath` also return `.init`. Should we file an issue for this?

Reply:
Yes